### PR TITLE
Adds new fat religion

### PR DESCRIPTION
**Pull request by Kharkov72**
The absolute unit religion!
Converting people to it makes them absolute units, the incense has a 5% chance to give you a small amount of nutriment (read: almost never).
Keywords include obesity, fat, obese, and absolute unit, and for reference, the incense stick is brown.
apple pie

:cl:
 * rscadd: Adds the fat religion.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -7432,3 +7432,16 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
 	description = "This has what you crave. Electrolytes."
 	sport = 5
 	custom_metabolism = 0.15
+
+/datum/reagent/incense/cornoil
+	name = "Corn Oil Incense"
+	id = INCENSE_CORNOIL
+	description = "This fragrance reminds you of a nice home-cooked meal, and sometimes even feels like it fills you up.."
```
Comment (DrSnips) [inline code/modules/reagents/Chemistry-Reagents.dm]: ..

**Comment by BomberBro** [comment]
taft

**Comment by Kurfursten** [comment]
I think @DeityLink had objections to increasing the number of non-flower based incenses, hence our compromise on plump helmets.

**Comment by DeityLink** [comment]
I guess we can make an exception for cornoil since it's got a very distinct smell.

**Review by Kurfursten — CHANGES_REQUESTED** [review]
You need to add a fragrance to the grown corn item, so that you can make more sticks

**Review by Kurfursten — APPROVED** [review]

**Pull request merged**